domains/elimcdonald.json

```
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "11Pixels",
+           "email": "",
+           "discord": "1058301993728356462",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.bZbUQ4FQSnByOLNg6JrNwTbYzco9jIHeXkBiKW2hZEi5UNp9JkiN0-eJwZgveTJTPT4DOYdZfL72Ch0dtAmgCqqbeO7pJiKBQ-ZoeJ2IJImLx1NnMLKhAjAGZArgdfNmkOlEheu88MTBIOAoMcUlOHBx2AwRhzVyUp0ikhLOfKXW76Q9IzSp7226jNcfh8kMdIyRuIvnZnljAbSYUq8zAkJ5A_uFqDjpli5pTZoJOeKhlgMQw-w_xZOBIYUwmDYNUNN_SnwhKlAjpeeFnIMED6GxxKO2lGnu-mqUPr3fhNuWgYOUOXC9EV9t8VwSLu3ZbEQ_PXGDn7No5qNWgn-0lg.c49GZQgJv3wHCHZdmFxd5w.ZPMC_7HPEnXmNJCXn9NVEE2SZVqju5TKM0G_pbplaHkU4NUOB5ZtmiP_CzQI-GYxaHrybIhKt7BsAbc5hKwiE2FqzEX0vWB10TCQ_GwlfiA.MmC33P5PRncvBWhLEaQXbw"
+        },
+    
+        "record": {
+            "CNAME": "185.199.109.153"
```

```suggestion
            "A": ["185.199.109.153"]
```
IPs should be A records.